CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ on GitHub if you don't have a token.
 
 All features should work, including building and launching.
 
-## Testing
+## Increase your GitHub API
```

Comment:
Do you mean increase your github api *limit*?

Reply:
Perhaps:

## Use a GitHub API token